src/Api.Rest.Dtos/Data/AttributeDto.cs

```diff
@@ -27,22 +27,16 @@ namespace Zeiss.PiWeb.Api.Rest.Dtos.Data
 	/// <remarks>This class is immutable.</remarks>
 	[Serializable]
 	[JsonConverter( typeof( AttributeConverter ) )]
-	public class AttributeDto : IEquatable<AttributeDto>

```

Comment:
Looks very good. I have added some basic tests for this class in PR #150.

Reply:
Perfect. Thank you :-)